### PR TITLE
Updates and additions

### PR DESCRIPTION
**Pull request by VladimirWork**

### CODE REVIEW

File: system/utils.py [hunk 1]
```diff
@@ -403,11 +472,21 @@ def check_no_failures(hosts):
...
-        retry_wait=20, timeout=480
+        check_primary_changed, pool_handler, wallet_handler, trustee_did, primary_before, retry_wait=20, timeout=480
     )
 
 
+async def check_pool_availability(pool_handle, wallet_handle, trustee_did):
+    req = await ledger.build_get_validator_info_request(trustee_did)
+    results = json.loads(await ledger.sign_and_submit_request(pool_handle, wallet_handle, trustee_did, req))
+    results = {k: json.loads(v) for k, v in results.items()}
+    assert all([v['result']['data']['Pool_info']['Unreachable_nodes_count'] == 0 for k, v in results.items()])
```
Comment (skhoroshavin) [inline system/utils.py]: Name of this check is a bit misleading in my opinion. Probably `check_all_nodes_connected` or `check_nodes_dont_have_connection_issues` or just `check_all_nodes_online` can be better.

File: system/aws_persistent_deployment.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+import time
+import botocore
+
+
+REGION_NAMES = [
+    'us-east-1', 'us-east-2', 'us-west-1', 'us-west-2', 'ap-northeast-2', 'ap-southeast-1', 'ap-southeast-2',
+    'ap-northeast-1', 'ca-central-1', 'eu-central-1', 'eu-west-1', 'eu-west-2', 'sa-east-1'
+]
+DEVICE = '/dev/sda1'
+OWNER = '962610246670'
```
Comment (skhoroshavin) [inline system/aws_persistent_deployment.py]: It's better to add comment stating that normally this ID shouldn't change so if someone creates a PR which tries to trick this script to use some 3rd party snapshot it becomes immediately obvious

Comment (VladimirWork) [reply system/aws_persistent_deployment.py]: Done.

File: system/utils.py (hunk 1 above)
Comment (VladimirWork) [reply system/utils.py]: Done.

File: system/utils.py [hunk 3]
```diff
@@ -54,6 +54,78 @@
     'RTZ3YMzzhsjeXkXieyhnvYdpLGBrEwcJWd3tDcX74EBTtghzW53DaVsYF4M5be6YBuLTGNiZkbzV7AhQ4gjtjsk8t6RFPPxxGtFxQMUNjPrRdJWwGrynTpRuk3vxWy1XKpmd5hEaauXNJdBLdj5cRFCae6WkqYTqbQN3kxpF3dd7cT'
 ]
 
+PERSISTENT_INSTANCES = {
+    'ap-northeast-1': ['i-07813055c26ecf5d2', 'i-0ba6de7b7e4ac763a'],
```
[the remark this answers is not in the document]
Comment (VladimirWork) [reply system/utils.py]: Done.

File: system/draft/test_misc.py [hunk 4]
```diff
@@ -1127,13 +1135,22 @@ def test_misc_hypothesis_sync(docker_setup_and_teardown, pool_handler, wallet_ha
 
 @pytest.mark.asyncio
 async def test_misc_is_1284():
```
Comment (ashcherbakov) [inline system/draft/test_misc.py]: It's not so clear what `is_1284` means and does. I think a test name with a short description and a more detailed description in the comment would be better.

Comment (VladimirWork) [reply system/draft/test_misc.py]: Done.

**Pull request merged**